strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -103,35 +88,6 @@ public Storage getStorage(String storageId)
         return configurationManager.getConfiguration().getStorage(storageId);
     }
 
-    @Override
```

Comment:
As discussed in skype `getInput/OutputStream` methods was reverted as `final` in `AbstractLayoutProvider`. And this is just for convenience.